app/assets/javascripts/discourse/app/routes/users.js

```diff
@@ -38,9 +39,12 @@ export default DiscourseRoute.extend({
   },
 
   model(params) {
-    const columns = PreloadStore.get("directoryColumns");
-    params.order = params.order || columns[0].name;
-    return { params, columns };
+    return ajax("directory-columns.json")
```

Comment:
The previous PR I did for this, used the preload store. This is now an ajax request